packages/documentation/src/stories/foundation/color/color-swatch.tsx

```diff
@@ -0,0 +1,19 @@
+export const ColorSwatch = (props: { color: string }) => (
+  <article className="color-square">
+    <div className={`color-square__inner bg-${props.color}`}>
+      <h3>{props.color}</h3>
```

Comment:
Why we just omit the color name here?
As it is allready documented with the variable names.

Reply:
For usability mainly and to show how text color changes based on background used.